src/main/java/org/kuali/kfs/gl/batch/service/impl/IndirectCostRecoveryGenerationMetadata.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2014 The Kuali Foundation
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.kuali.kfs.gl.batch.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.cornell.kfs.gl.batch.service.impl.CuIndirectCostRecoveryAccountDistributionMetadata;
+
+public class IndirectCostRecoveryGenerationMetadata {
```

Comment:
`List<IndirectCostRecoveryAccountDistributionMetadata> accountLists` is private. Would I create a new private class and override the the getters/setters then? I didn't think that would be the best course of action because accountLists is used in various class methods